agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/AiConfigCustomizer.java

```diff
@@ -139,7 +139,7 @@ private static void enableInstrumentations(
     properties.put("otel.instrumentation.apache-httpasyncclient.enabled", "true");
     properties.put("otel.instrumentation.apache-httpclient.enabled", "true");
     properties.put("otel.instrumentation.async-http-client.enabled", "true");
-    properties.put("otel.instrumentation.executor.enabled", "true");
+    properties.put("otel.instrumentation.executors.enabled", "true");
```

Comment:
ugh, I thought this was enabled upstream by default (even under default-enabled=false)

also pretty surprised only one of our smoke tests failed without it